code/datums/votes/map_vote.dm

```diff
@@ -66,10 +66,11 @@
 	message = initial(message)
 	return TRUE
 
-/// Before we create a vote, remove all maps from our choices that are outside of our population range. Note that this can result in zero remaining choices for our vote, which is not ideal. Let's warn users about that if it comes up.
-/// Argument create_vote is typically FALSE, pass as TRUE if calling from the create_vote proc.
+/// Before we create a vote, remove all maps from our choices that are outside of our population range. Note that this can result in zero remaining choices for our vote, which is not ideal (but ultimately okay).
+/// Argument create_vote is typically FALSE, pass as TRUE if calling from the create_vote proc. When FALSE, we collect the default choices in a similar manner to how the create_vote() proc does it.
```

Comment:
you don't need to detail where it comes from. you almost shouldn't, since it could change outside your control. it's better to describe JUST what it does, or why you might want to do that. yaknow?
you could also just split this into two procs, one that calls the other, which might make things a bit cleaner